docs/README.md

```diff
@@ -1846,6 +1862,188 @@ boost nuke -e production
 
 **Note**: This will delete everything in your stack, including databases. This action is **not** reversible!
 
+## Going deeper with Booster
+
+### Framework Core
+
+### Framework Types
+
+### Framework integration tests
+
+Booster framework integration tests package is used to test the Booster project itself, but it is also an example of how a Booster application could be tested. We encourage developers to have a look at our [Booster project repository](https://github.com/boostercloud/booster/tree/master/packages/framework-integration-tests).
+
+Some integration tests highly depend on the provider chosen for the project, and the infrastructure is normally deployed locally or in the cloud right before the tests run. Once tests are completed, the application is teared down.
+
+There are several types if integration tests in this package:
+- Tests to ensure that different packages integrate as expected with each other
+- Tests to ensure that a Booster application behaves as expected when it is hit by a client (a GraphQL client)
+- Tests to ensure that the application behaves in the same way no matter what provider is selected
+
+If you are curious about the framework providers, you will be able to read more about them in the following section.
+
+### Providers
+
+The providers are the different implementations of Booster backend depending on the cloud provider or technologies. They all implement the same interface, and the main idea behind the providers is that no matter what the developer chooses as backend, they will not need to know anything about infrastructure.  
+
+#### framework-provider-aws-*
+
+#### framework-provider-local-*
+
+The Booster framework local provider combines in-memory databases with a GraphQL API served through a Node.js Express Server.
+
+One of the advantages of having a local provider available is the fact that the deployment time dramatically drops, being able to speed up the development time.
+
+#### framework-provider-kubernetes-*
+
+#### framework-provider-azure-*
+
+### Configuration and environments
+
+One of the goals of booster is to avoid configuration parameters as much as possible, inferring everything automatically from your code. However, there are some aspects that can't be inferred (like the application name), or it is useful to change them under certain circumstances, especially when using different [environments](#environments)
+
+#### Booster configuration
+```typescript
+import { Booster } from '@boostercloud/framework-core'
+import { BoosterConfig } from '@boostercloud/framework-types'
+import { Provider } from '@boostercloud/framework-provider-aws'
+
+Booster.configure('pre-production', (config: BoosterConfig): void => {
+  config.appName = 'my-app-name'
+  config.provider = Provider
+})
```

Comment:
Perhaps using a qualified import like you suggested?

```suggestion
import { Booster } from '@boostercloud/framework-core'
import { BoosterConfig } from '@boostercloud/framework-types'
import AWS from '@boostercloud/framework-provider-aws'

Booster.configure('pre-production', (config: BoosterConfig): void => {
  config.appName = 'my-app-name'
  config.provider = AWS.Provider
})
```

Reply:
done, thanks @NickSeagull e1eb2fe74d3221c758e369332a162aab80e67913